tests/Unit/PointwiseFunctions/Xcts/Test_AdmMass.cpp

```diff
@@ -171,23 +159,46 @@ void test_mass_surface_integral(const double distance,
           surface_integrand(ti::I) * conformal_face_normal(ti::i));
 
       // Compute contribution to surface integral
-      surface_integral += definite_integral(
+      surface_integral.get() += definite_integral(
           get(contracted_integrand) * get(conformal_area_element), face_mesh);
     }
   }
 
   // Check result
   auto custom_approx = Approx::custom().epsilon(10. / distance).scale(1.0);
-  CHECK(surface_integral == custom_approx(lorentz_factor * mass));
+  const double lorentz_factor = 1. / sqrt(1. - square(boost_speed));
+  CHECK(get(surface_integral) == custom_approx(lorentz_factor * mass));
 }
 
 }  // namespace
 
 SPECTRE_TEST_CASE("Unit.PointwiseFunctions.Xcts.AdmMass",
                   "[Unit][PointwiseFunctions]") {
-  const size_t P = 6;
-  const size_t L = 1;
-  for (double distance : std::array<double, 3>{{1.e3, 1.e5, 1.e10}}) {
-    test_mass_surface_integral(distance, L, P);
+  // Define black hole parameters.
```

Comment:
Use scopes like this:

```cpp
{
  INFO("Boosted Kerr-Schild");
  const double mass = 1;
  // ...
}
{
  INFO("Isotropic Schwarzschild");
  // ...
}
```

Then you can use the same variable names in both sections, and you get more information when a test fails (the `INFO` is printed).